tests/test-list-ubuntu18.04.golden.output

```diff
@@ -0,0 +1,7 @@
++-----------------------------+----------+-------+-------------------------------+-----+---------+---------+
```

Comment:
Right now, considering how the golden output works, I can only test for non running containers, since the running ones have dynamic data. Like the host port number which would be different on each test run. This is good enough for now...

Reply:
I'm envisioning being able to query parts of the json like `docker inspect`:

```
footloose show node0 -f '{{.Status}}'
```
we could use to test `show` when we have that.